cdap-common/pom.xml

```diff
@@ -214,6 +214,17 @@
       <version>1.8.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
```

Comment:
why do these need to be added to cdap-common?

Reply:
twill was bringing kafka for scala 2.10. I excluded it as it's not compatible with 2.11/2.12 and added 2.11 explicitly when needed.